skeletonkey/instantiate.py

```diff
@@ -23,6 +24,45 @@ def import_target(class_string: str) -> Type[Any]:
     return obj
 
 
+def partial_instantiate(config, target_keyword=TARGET_KEYWORD,_instantiate_recursive=True, **kwargs) -> Any:
```

Comment:
Logan says he thinks that the function should specifically be instantiate_partial to match the formatting of instantiate_all.

Reply:
Gotcha, yeah sorry about that, it should be changed now I think! Sorry I think I made that edit while you were making the comment lol

Reply:
Epic